src/kochka.com.mx/.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
IMHO all the `src/kochka.com.mx/.github/*` files make no sense here

Reply:
You are right. 👍 A relic from the private repo.